lib/vehicle_factory.rb

```diff
@@ -0,0 +1,16 @@
+class VehicleFactory
+
+  def create_vehicles(vehicles)
+    vehicles.map do |vehicle|
+      vehicle_details = {
+        vin: vehicle[:vin_1_10],
+        year: vehicle[:model_year],
+        make: vehicle[:make],
+        model: vehicle[:model],
+        engine: :ev,
+        county: vehicle[:county]
+      }
+      Vehicle.new(vehicle_details)
```

Comment:
Same thing here as the FacilitiesFactory. Would be helpful to have an instance variable declared in initialize that you could feed all these instances into so you only have to run this map once. Great job with the map!